pod.go

```diff
@@ -383,6 +390,16 @@ func createPod(podConfig PodConfig) (*Pod, error) {
 		state:      State{},
 	}
 
+	// Create network only if the namespace is empty in our configuration.
+	// Being empty means this is the first time we create the network for
+	// this pod.
+	if podConfig.NetworkConfig.NetworkNSPath == "" {
```

Comment:
I get it, but we don't want to call network creation every time we are in createPod() because createPod() is called from fetchPod(), meaning it is called from every API.
Maybe we could call it from CreatePod() in api.go instead ?